src/sentry/api/endpoints/project_group_index.py

```diff
@@ -27,6 +28,7 @@ class ProjectGroupIndexEndpoint(ProjectEndpoint, EnvironmentMixin):
     permission_classes = (ProjectEventPermission,)
 
     @track_slo_response("workflow")
+    @rate_limit_endpoint(limit=3, window=1)
```

Comment:
didn't even know we had a decorator for this.

Is the window in seconds or min?

Reply:
It's seconds. So this is 3 requests/second/ip address

Reply:
👍 lgtm

Reply:
Do the limit/window values need to be in config so we don't impact open source?

Reply:
Maybe at some point, these limits are pretty high so I don't think we'll impact anyone right now. I'd say we probably just want to entirely disable rate limiting for open source, so maybe just a bool to handle that.